opendbc/car/hyundai/values.py

```diff
@@ -745,6 +752,8 @@ def match_fw_to_car_fuzzy(live_fw_versions, vin, offline_fw_versions) -> set[str
 
 EV_CAR = CAR.with_flags(HyundaiFlags.EV)
 
+FCEV_CAR = CAR.with_flags(HyundaiFlags.FCEV)
```

Comment:
This is redundant, it can be removed along with the other places it appears.

Think we can probably ditch `EV_CAR` too, but haven't fully checked, and that's a separate PR anyway.